### PR TITLE
claimsmanager: improve test coverage

### PR DESCRIPTION
**Pull request by pysel**
## 1. Summary
Fixes # (issue)

## 2.Type of change

Improves test coverage of `claimsmanager` module to 81%

### CODE REVIEW

File: x/claimsmanager/keeper/self_consensus_state_test.go [hunk 1]
```diff
@@ -17,4 +17,9 @@ func (suite *KeeperTestSuite) TestGetSetDelete() {
 
 	_, found = k.GetSelfConsensusState(ctx, "test")
 	suite.Require().False(found)
+
+	err = k.StoreSelfConsensusState(ctx, "test-1")
```
Comment (joe-bowman) [inline x/claimsmanager/keeper/self_consensus_state_test.go:21]: err should be checked with `suite.NoError(err)`

**Review by joe-bowman — CHANGES_REQUESTED** [review]
check value of err please.

**Comment by pysel** [comment]
hey @joe-bowman , error checked

**Review by joe-bowman — APPROVED** [review]
lgtm

**Pull request merged**